src/pages/index.jsx

```diff
@@ -1,20 +1,72 @@
-import React from 'react';
-import FancyDiv from '../components/FancyDiv';
+import React, { Fragment } from 'react';
+import ReactPlayer from 'react-player';
+import { Table } from 'reactstrap';
+import logo from '../assets/images/ITExpo.jpg';
 
 const IndexPage = () =>
-  <div className="default-content">
-    <div className="content">
-      <FancyDiv>
+  <Fragment>
+    <div className="default-content">
+      <div className="content">
+
         <div style={{ textAlign: `center` }}>
-          <h1>
-            Welcome to React-Static <br />
-          </h1>
-          <p>
-            Start Here
-          </p>
+          <img src={logo} alt='IT Expo' />
+          <h2>UC School of Information Technology</h2>
+        </div>
+
+        <div class="container">
+          <div class="row">
+            <div class="col-sm-9">
+              <h1>Schedule </h1>
```

Comment:
```suggestion
              <h2>Schedule </h2>
```
Each page should only have 1 `<h1>` element for accessibility reasons and then the next heading level would be `<h2>`